DirectProgramming/DPC++FPGA/ReferenceDesigns/decompress/src/CMakeLists.txt

```diff
@@ -33,8 +33,20 @@ else()
   set(ignoreMe "${GZIP}${SNAPPY}")
 endif()
 
-# Include ac_types headers and link against ac_types emulation libraries
-set(AC_TYPES_FLAG "-qactypes")
+# These are Windows-specific flags:
+# 1. /EHsc This is a Windows-specific flag that enables exception handling in host code
+# 2. /Qactypes Include ac_types headers and link against ac_types emulation libraries
+if(WIN32)
+    set(WIN_FLAG "/EHsc")
+    set(AC_TYPES_FLAG "/Qactypes")
+    # Increase the allowable constexpr steps for the front end. This allows the
+    # front-end compiler to do more compile-time computation.
+    set(CONSTEXPR_STEPS "/constexpr:steps5084968")
```

Comment:
The test fails for another reason (default selector) so I can't really tell if this flag is working or not but I didn't get a warning

Reply:
This looks off to me though. Can you check with the test owner to confirm? This shouldn't block the check-in though since this change is unblocking other issues.